### DISCUSSION TITLE
View Environment Details Not Matching Setup>Company Information Org Id

### DISCUSSION THREAD

**Discussion by shuylern** [jongpie/NebulaLogger#818] [Q&A] [answered]
Has anyone seen an issue where the Nebula > View Environment Details Org Id value does not match the Org Id value in the Setup > Company Information panel?  

Just wondering....

**Comment by jongpie** [comment]
@shuylern can you share a screenshot of what you're seeing in Nebula Logger vs Company Info?

**Comment by shuylern** [comment, marked as answer]
@jongpie  > It was user error (mine).  I had too many tabs open with different Orgs.  Sorry for the confusion.

**Reply by jongpie** [reply]
No worries at all!
